bugbug/models/regressor.py

```diff
@@ -121,9 +122,11 @@ def __init__(
                 ("union", ColumnTransformer(column_transformers)),
             ]
         )
+        base_clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+        base_clf.set_params(predictor="cpu_predictor")
+        self.clf = IsotonicRegressionCalibrator(base_clf)
 
-        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
-        self.clf.set_params(predictor="cpu_predictor")
+        self.calculate_importance = False
```

Comment:
Is `calculate_importance = False` still needed?

Reply:
lemme try it without that!

Reply:
its needed! otherwise it gives the precious error! maybe we can raise an issue regarding this?